presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -878,7 +878,7 @@ public void testViewAccessControl()
     {
         skipTestUnless(supportsViews());
 
-        Session viewOwnerSession = TestingSession.testSessionBuilder()
```

Comment:
While this refactoring change is fine, it's unrelated to the rest of this commit, so please remove it.